packages/web/app/components/Field/Field.jsx

```diff
@@ -1,134 +1,2 @@
-  }) => {
-    // Only show error messages once the user has attempted to submit the form
-    const error = submitStatus === FORM_STATUSES.SUBMIT_ATTEMPTED && !!getIn(errors, name);
-    const message = error ? getIn(errors, name) : helperText;
-
-    const { setFieldTouched } = useFormikContext();
-    const [field] = useField(name);
-    const fieldValue = getIn(values, name);
-
-    const isFormSubmitted = useRef(false);
-
-    // Set a flag to indicate that the form has been submitted
-    useEffect(() => {
-      if (submitStatus === FORM_STATUSES.SUBMIT_ATTEMPTED) {
-        setTimeout(() => {
-          isFormSubmitted.current = true;
-        });
-      }
-    }, [submitStatus]);
-
-    // Validate field when its value changes (only after submit attempt and if there is an error)
-    useEffect(() => {
-      if (error && isFormSubmitted.current) {
-        validateField(name);
-      }
-    }, [fieldValue]);
-
-    const baseOnChange = (...args) => {
-      setFieldTouched(name, true);
-      return field.onChange(...args);
-    };
-
-    const combinedOnChange = onChange
-      ? (...args) => {
-          onChange(...args);
-          return baseOnChange(...args);
-        }
-      : baseOnChange;
-
-    return (
-      <FormikField
-        {...props}
-        component={component}
-        error={error}
-        helperText={message}
-        name={name}
-        onChange={combinedOnChange}
-      />
-    );
-  },
-);
-
-/**
- * A formik form field with an added tooltip
- *
- * @param tooltipText - the text for the tooltip to show
- * @param disabledTooltipText = the text for the tooltip to show when the field is disabled
- * @param muiTooltipProps - material ui tooltip props @see https://v4.mui.com/api/tooltip
- *
- */
-const StyledToolTip = styled(ThemedTooltip)`
-  .MuiTooltip-tooltip {
-    top: 30px !important;
-    font-weight: 400;
-    text-align: center;
-  }
-`;
-export const FieldWithTooltip = ({
-  tooltipText,
-  disabledTooltipText,
-  muiTooltipProps,
-  ...props
-}) => {
-  if (disabledTooltipText && props.disabled)
-    return (
-      <MuiBox position="relative" data-testid="muibox-slpq">
-        <StyledToolTip
-          title={disabledTooltipText}
-          arrow
-          placement="top"
-          {...props}
-          data-testid="styledtooltip-qc9k"
-        >
-          {/* Below div is needed to make StyledToolTip work  */}
-          <div>
-            <Field {...props} data-testid="field-ete4" />
-          </div>
-        </StyledToolTip>
-      </MuiBox>
-    );
-
-  return (
-    <MuiBox position="relative" data-testid="muibox-8z4o">
-      <Field {...props} data-testid="field-tybt" />
-      {tooltipText && (
-        <FormTooltip
-          arrow
-          placement="top"
-          title={tooltipText}
-          {...muiTooltipProps}
-          data-testid="formtooltip-lztn"
-        />
-      )}
-    </MuiBox>
-  );
-};
+import { Field, FieldWithTooltip } from '@tamanu/ui-components';
```

Comment:
@dannash100 I have done the re-export pattern here. I'm not sure how to keep track of these?